extensions/amp-mustache/0.1/amp-mustache.js

```diff
@@ -58,8 +58,10 @@ export class AmpMustache extends AMP.BaseTemplate {
 
   /** @override */
   render(data) {
-    const html = mustacheRender(this.template_,
-        Object.assign({}, data, this.nestedTemplates_));
+    if (typeof data !== 'string') {
```

Comment:
how about numbers, booleans, arrays? should this be `if(typeof data === 'object')` instead?
/cc @cvializ

Reply:
haha you fixed it as I was writing this :) do you have mind-reading powers?

Reply:
🔮